migrating_to_2.0/recipes.rst

```diff
@@ -30,6 +30,8 @@ Python import statements
     from conan.tools.files import save, load
     from conan.tools.gnu import AutotoolsToolchain, AutotoolsDeps
     from conan.tools.microsoft import unix_path, VCVars, is_msvc
+    from conan.errors import ConanInvgoialidConfiguration
+    from conan.errors import ConanException
```

Comment:
Are we missing

```python
from conans.errors import ConanInvalidSystemRequirements
```
?

Reply:
I don't think it is needed to document here all the possible imports but some examples